op-e2e/system/fees/eip1559params_test.go

```diff
@@ -0,0 +1,105 @@
+	"github.com/stretchr/testify/require"
+)
+
+// TestEIP1599Params checks that we can successfully change EIP-1559 parameters via SysConfig with
+// the Holocene upgrade.
+func TestEIP1559Params(t *testing.T) {
+	op_e2e.InitParallel(t)
+
+	ctx, ctxCancel := context.WithCancel(context.Background())
+	defer ctxCancel()
+
+	// Create our system configuration for L1/L2 and start it
+	cfg := e2esys.HoloceneSystemConfig(t, new(hexutil.Uint64))
+	cfg.DeployConfig.L2GenesisBlockBaseFeePerGas = (*hexutil.Big)(big.NewInt(100_000_000))
+	sys, err := cfg.Start(t)
+	require.NoError(t, err, "Error starting up system")
+
+	// Obtain our sequencer, verifier, and transactor keypair.
+	l1Client := sys.NodeClient("l1")
+	l2Seq := sys.NodeClient("sequencer")
+	ethPrivKey := cfg.Secrets.SysCfgOwner
+
+	_, err = l2Seq.HeaderByNumber(context.Background(), big.NewInt(0))
+	require.NoError(t, err)
+
+	// Bind to the SystemConfig contract
+	sysconfig, err := legacybindings.NewSystemConfig(cfg.L1Deployments.SystemConfigProxy, l1Client)
+	require.NoError(t, err)
+
+	// Obtain our signer.
+	opts, err := bind.NewKeyedTransactorWithChainID(ethPrivKey, cfg.L1ChainIDBig())
+	require.NoError(t, err)
+
+	// Define our L1 transaction timeout duration.
+	txTimeoutDuration := 10 * time.Duration(cfg.DeployConfig.L1BlockTime) * time.Second
+
+	var cancel context.CancelFunc
+
+	// confirm eip-1559 parameters are initialized to 0
+	denom, err := sysconfig.Eip1559Denominator(&bind.CallOpts{})
+	require.NoError(t, err, "reading denominator")
+	require.Equal(t, uint32(0), denom)
+
+	elasticity, err := sysconfig.Eip1559Elasticity(&bind.CallOpts{})
+	require.NoError(t, err, "reading elasticity")
+	require.Equal(t, uint32(0), elasticity)
+
+	// update the EIP-1559 params, wait for it to show up on L2, & verify that it was set as intended
+	expectedDenom := uint32(10)
+	expectedElasticity := uint32(2) // gas target will be 15_000_000
+	opts.Context, cancel = context.WithTimeout(ctx, txTimeoutDuration)
+	tx, err := sysconfig.SetEIP1559Params(opts, expectedDenom, expectedElasticity)
+	cancel()
+	require.NoError(t, err, "SetEIP1559Params update tx")
+
+	receipt, err := wait.ForReceiptOK(ctx, l1Client, tx.Hash())
+	require.NoError(t, err, "Waiting for sysconfig set gas config update tx")
+
+	denom, err = sysconfig.Eip1559Denominator(&bind.CallOpts{})
+	require.NoError(t, err, "reading denominator")
+	require.Equal(t, expectedDenom, denom)
+
+	elasticity, err = sysconfig.Eip1559Elasticity(&bind.CallOpts{})
+	require.NoError(t, err, "reading elasticity")
+	require.Equal(t, expectedElasticity, elasticity)
+
+	_, err = geth.WaitForL1OriginOnL2(sys.RollupConfig, receipt.BlockNumber.Uint64(), l2Seq, txTimeoutDuration)
+	require.NoError(t, err, "waiting for L2 block to include the sysconfig update")
+
+	h, err := l2Seq.HeaderByNumber(context.Background(), nil)
+	require.NoError(t, err)
+
+	// confirm the extraData is being set as expected
+	eip1559.DecodeHoloceneExtraData(h.Extra)
```

Comment:
This skips assigning the return values to make sure it doesn't panic?

Reply:
You could then wrap it in a `require.NoPanic`.